src/seml/add.py

```diff
@@ -95,25 +109,59 @@ def add_configs(collection, seml_config, slurm_config, configs, source_files=Non
 
     if source_files is not None:
         seml_config['source_files'] = source_files
-    db_dicts = [{'_id': start_id + ix,
+    db_dicts = [{'_id': start_id + idx,
                  'batch_id': batch_id,
                  'status': States.STAGED[0],
                  'seml': seml_config,
                  'slurm': slurm_config,
                  'config': c,
-                 'config_hash': make_hash(c),
+                 'config_unresolved' : c_unresolved,
+                 'config_hash': make_hash(c, config_get_exclude_keys(c, c_unresolved)),
                  'git': git_info,
                  'add_time': datetime.datetime.utcnow()}
-                for ix, c in enumerate(configs)]
+                for idx, (c, c_unresolved) in enumerate(zip(configs, configs_unresolved))]
+    if resolve_descriptions:
+        for db_dict in db_dicts:
+            if 'description' in db_dict['seml']:
+                db_dict['seml']['description'] = resolve_description(db_dict['seml']['description'], db_dict['config'])
```

Comment:
Th only downside I see is that you'd need to specify `config.foo.bar` instead of `foo.bar`. This would in a way however be more consistent with what the `-p` flag does. So we could also implement it that way.

Reply:
I'd argue for consistency with `-p`

Reply:
Agreed, will resolve against the entire document.